snippets/cnf-clusterinstance-cr.adoc

```diff
@@ -0,0 +1,168 @@
+.Reference ClusterInstance CR
+[source,yaml]
+----
+apiVersion: siteconfig.open-cluster-management.io/v1alpha1
+kind: ClusterInstance
+metadata:
+  annotations:
+    kubectl.kubernetes.io/last-applied-configuration: |
+      <last_applied_configuration>
+  creationTimestamp: "2024-07-01T09:00:53Z"
+  finalizers:
+    - clusterinstance.siteconfig.open-cluster-management.io/finalizer
+  generation: 2
+  name: example-cluster
+  namespace: example-cluster
+  resourceVersion: "<resource_version>"
+  uid: <uid>
+spec:
+  additionalNTPSources:
+    - <NTP_resource>
+  baseDomain: <base_domain>
+  clusterImageSetNameRef: <cluster_image_set_reference>
+  clusterLabels:
+    common: "true"
+    sites: example-cluster
+  clusterName: example-cluster
+  clusterNetwork:
+    - cidr: <cidr>
+      hostPrefix: <host_prefix>
+  cpuPartitioningMode: None
+  extraAnnotations:
+    ClusterDeployment:
+      myTestAnnotation: success
+  holdInstallation: false
+  installConfigOverrides: '<user_overrides>'
+  machineNetwork:
+    - cidr: <cidr>
+  networkType: OVNKubernetes
+  nodes:
+    - automatedCleaningMode: disabled
+      bmcAddress: <BMC_address>
+      bmcCredentialsName:
+        name: <BMC_secret_name>
+      bootMACAddress: <BMC_MAC_address>
+      bootMode: UEFI
+      hostName: example-cluster.example.redhat.com
+      ironicInspect: ""
+      nodeNetwork:
+        config:
+          <network_configuration>
+        interfaces:
+          <interfaces>
+      role: master
+      templateRefs:
+        - name: ai-node-templates-v1
+          namespace: siteconfig-system
+  pullSecretRef:
+    name: pull-secret
+  serviceNetwork:
+    - cidr: <cidr>
+  sshPublicKey: <public_ssh_key>
+  templateRefs:
+    - name: ai-cluster-templates-v1
+      namespace: siteconfig-system
+status:
```

Comment:
The status is not something the user configures so can remove everything from line 65 onwards